### PR TITLE
initial docker resources for uds-agent

### PR DESCRIPTION
**Pull request by monalabs**
This is the initial docker resources for mona-uds-agent. README is WIP.
@nemo-mona

### CODE REVIEW

File: agent/README.md [hunk 1]
```diff
@@ -1,4 +1,65 @@
-# Mona Agent
+# Mona UDS Agent
 
-Mona agent
-Todo(Tzach): Complete this
+### Mona-UDS-Agent docker image
+
+This docker image runs mona-uds-agent, built on top of FluentD Technology.
```
Comment (itaiMona) [inline agent/README.md]:
```suggestion
This docker image runs mona-uds-agent, built on top of FluentD.
```

File: agent/README.md [hunk 2]
```diff
@@ -1,4 +1,65 @@
...
+
+##### You should use mona-uds-agent when:
+1. You signed up on https://www.monalabs.io/getstarted and have a private endpoint to send your data to Mona
+2. Your models run on a docker orchestration such as Kubernetes and you wish to send their outputs directly to Mona
```
Comment (itaiMona) [inline agent/README.md]:
```suggestion
2. Your system runs on a docker orchestration platform such as Kubernetes and you wish to export data  directly from it to Mona
```

File: agent/README.md [hunk 3]
```diff
@@ -1,4 +1,65 @@
...
+3. You want to send your data through Unix Domain Sockets
+
+###### How to run it
+
+The following environment variables are required:
+
+* MONA_USER_ID: your user id (uuid format)
```
Comment (itaiMona) [inline agent/README.md]:
```suggestion
* MONA_USER_ID: the user-id provided to you by Mona (can be found on dashboard.monalabs.io/configurations)
```

File: agent/README.md [hunk 4]
```diff
@@ -1,4 +1,65 @@
...
+
+You can use the following YAML formatted file on Kubernetes. 
+UDS agents have to be on all nodes, that's why it's required to use DaemonSet
```
Comment (itaiMona) [inline agent/README.md]: They're required to be on "all" nodes, or on all nodes from which data needs to be sent?

File: agent/README.md [hunk 5]
```diff
@@ -1,4 +1,65 @@
...
+      containers:
+        - name: mona-uds-agent
+          imagePullPolicy: Always
+          image: monalabs/mona-uds-agent:stable
+          env:
+            - name: MONA_USER_ID
+              value: '<YOUR_PRIVATE_MONA_USER_ID>'
+          resources:
+            limits:
+              cpu: '2'
```
Comment (itaiMona) [inline agent/README.md:56]: Can these limits be played with as the user wishes? 

I would expect to have some details on:
1. Which yml fields (if any) are expected to remain exactly the same or the agent to function properly
2. If there are yaml fields (if any) which must have specific value ranges (e.g., never provide less than 1 cpu for this...)
3. Which yaml fields (if any) are expected to change (e.g., "we use 2 cpus by default as limit, but feel free to change this between the values X and Y") - you gave the user id example for this - so maybe you thought of this already :)

**Review by itaiMona — COMMENTED** [review]
Thanks Tzach! Cool stuff :)

File: agent/README.md (hunk 5 above)
Comment (monalabs) [reply agent/README.md:56]: Added some more info under the YAML.

**Pull request merged**